app/services/knowledge_base/ingestion.py

```
@@ -0,0 +1,212 @@
+        changed: List[tuple[int, PreparedChunk]] = [
+            (index, chunk)
+            for index, chunk in enumerate(chunks)
+            if existing_hashes.get(index) != chunk.content_hash
+        ]
+
+        embedded = 0
+        for batch in _embedding_batches(changed):
+            texts = [chunk.text for _, chunk in batch]
+            provider = get_embedding_provider(kb.embedding_config)
+            vectors = await provider.embed(texts, input_type="document")
+            for (index, chunk), vector in zip(batch, vectors):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)